app/views/categories/index.html.erb

```diff
@@ -0,0 +1,66 @@
+<div>
+  <nav class="navbar navbar-expand-lg navbar-dark bg-navy">
+    <div class="container">
+      <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarTogglerDemo03" aria-controls="navbarTogglerDemo03" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <a class="navbar-brand" href="#">Categories</a>
+      <div class="collapse navbar-collapse" id="navbarTogglerDemo03">
+        <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+          <% if current_user %>
+            <li class="nav-item mt-3">
+              <%= link_to "Hi, #{current_user.email}", edit_user_registration_path, class: "nav-link" %>
+            </li>
+            <li class="nav-item mt-3 signout-btn">
+              <%= button_to "Sign Out", destroy_user_session_path, method: :delete, class: "btn btn-outline-danger" %>
+            </li>
+          <% end %>
+        </ul>
+      </div>
+    </div>
+  </nav>
```

Comment:
- [ OPTIONAL] I highly recommend making the button for adding a new category and adding a new transaction have a green background and a width the same in the given wireframe so please make them the same but I am assigning it as optional because the previous reviewer didn't mention that but it still important to do it ♻🙏🏻✅⭐ 

![image](https://github.com/Surafels/budget-app/assets/58553711/d0c95b7a-bd67-4bd9-b2b6-2897c800aec2)